tests/operations.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { test } from "./fixtures/lxd-test";
+import {
+  createInstance,
+  deleteInstance,
+  randomInstanceName,
+  visitAndStartInstance,
+  visitAndStopInstance,
+} from "./helpers/instances";
+import {
+  createProfile,
+  deleteProfile,
+  randomProfileName,
+} from "./helpers/profile";
+import { assertTextVisible } from "./helpers/permissions";
+import {
+  checkNotificationExists,
+  dismissNotification,
+} from "./helpers/notification";
+
+let instance = randomInstanceName();
+let profile = randomProfileName();
+
+test.beforeAll(async ({ browser, browserName }) => {
+  instance = `${browserName}-${instance}`;
+  profile = `${browserName}-${profile}`;
+  const page = await browser.newPage();
+  await createProfile(page, profile);
+  await createInstance(page, instance);
+  await page.close();
+});
+
+test.afterAll(async ({ browser }) => {
+  const page = await browser.newPage();
+  await deleteProfile(page, profile);
+  await page.close();
+});
+
+test("instance operations are recognised on the Operations page", async ({
+  page,
+}) => {
+  await page.goto("/ui/operations");
+
+  // validate create operation is in operation list
+  await page
+    .getByRole("row", { name: "Action" })
+    .filter({ hasText: "Starting Instance" })
+    .filter({ hasText: instance })
+    .getByText(instance);
```

Comment:
```suggestion
  const operationLocator = await page
    .getByRole("row", { name: "Action" })
    .filter({ hasText: "Starting Instance" })
    .filter({ hasText: instance })
    getByText(instance);
   await expect(operationLocator).toBeVisible();
```
I think we need the extra expect. Might be worth to put this into a helper function like `validateOperation(title:string, instance: string)` and reuse it multiple time in this test. With that, the code might get very straight forward and reduce the need for some comments.

Reply:
Playwright/code doesn't like this. I get an infinite loading screen when using the trace viewer, and when using the browser although the element is there, it isn't being picked up on. I also received this error:
![image](https://github.com/canonical/lxd-ui/assets/58699506/178eb911-6e91-4311-a5e7-8bc250cbee63)Any thoughts on how to get around this?

Reply:
@edlerd although I have commented out the usage in the latest commit, you can find the implementation of the helper function at tests/helpers/operations.ts.

Reply:
@Kxiru in vscode for the playwright extension, you can use the "Pick locator" functionality to identify the selector required for targeting a specific element in the test browser. See if that helps with the above error?

![Screenshot from 2024-02-21 13-09-57](https://github.com/canonical/lxd-ui/assets/151511568/5b974b7c-d25c-4ddc-92a5-9c6d324dc245)